### PR TITLE
New package: Basti-def.helloworld.package version 1.0

### PR DESCRIPTION
**Pull request by bxsti-dev**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151064)

### CODE REVIEW

**Comment by Basti-def** [comment]
@microsoft-github-policy-service agree [company="Basti-def"]

**Comment by Basti-def** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Verify: 
image

**Comment by stephengillie** [comment]
Manual Validation ended with:
> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\helloworld.exe    ←[1m←[31m      H←[0m ←[1m←[91me←[0m ←[1m←[33ml←[0m ←[1m←[32ml←[0m ←[1m←[34mo←[0m ←[1m←[97m  ←[0m ←[1m←[36mW←[0m ←[1m←[35mo←[0m ←[1m←[91mr←[0m ←[1m←[33ml←[0m ←[1m←[32md←[0m ←[1m←[34m!←[0m    PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Basti-def.helloworld.package__DefaultSource\helloworld.exe    ←[1m←[31m      H←[0m ←[1m←[91me←[0m ←[1m←[33ml←[0m ←[1m←[32ml←[0m ←[1m←[34mo←[0m ←[1m←[97m  ←[0m ←[1m←[36mW←[0m ←[1m←[35mo←[0m ←[1m←[91mr←[0m ←[1m←[33ml←[0m ←[1m←[32md←[0m ←[1m←[34m!←[0m    PS C:\Users\User\Desktop>

(Automated response - build 877.)

**Comment by Basti-def** [comment]
Hi,
I don't know if colors are displayed correctly here on reddit. 
But the Text (Hello World!) should have colors.

**Comment by stephengillie** [comment]
Is the application supposed to display `←[1m←[31m`? You said that colors should be displayed instead. Sorry if I haven't been clear, but the bits pasted into the above comment were identical with the console display - all in default white-on-blue, not a series of colors.

**Comment by Basti-def** [comment]
I tested it on a virtual machine. 
You were right, I get the same output when I run this with the old powershell or cmd. 
It only works in the newer "Terminal" App.

I guess it have to work on both?

Well actually, this is more like a "demo" project, to test if I can upload to winget. 
Can I close the commit, delete the repo and create a real + bigger project later, and start over?

If that's all I have to do, I have one more question: If I ever wanted to delete a winget project... Would it be enough to just delete the repo?

Thank you for understanding

**Review by stephengillie — CHANGES_REQUESTED** [review]
Oh - with the dependency, it operates as expected:
image

File: manifests/b/Basti-def/helloworld/package/1.0/Basti-def.helloworld.package.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Basti-def.helloworld.package
+PackageVersion: "1.0"
+InstallerType: portable
+Commands:
+- helloworld
+Installers:
```
Comment (stephengillie) [inline manifests/b/Basti-def/helloworld/package/1.0/Basti-def.helloworld.package.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.WindowsTerminal
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
(This was supposed to be in the previous review, but I forgot to hit "Start review".)

**Comment by Basti-def** [comment]
Hi,
so that means when I add this dependency in furture projects, it works without issues?

I answerd you last time but the bot didn't do anything. (remove/add labels,  assigne users, ..)
So I send it here again:

>I tested it on a virtual machine.
You were right, I get the same output when I run this with the old powershell or cmd.
It only works in the newer "Terminal" App.
I guess it have to work on both?
Well actually, this is more like a "demo" project, to test if I can upload to winget.
Can I close the commit, delete the repo and create a real + bigger project later, and start over?
If that's all I have to do, I have one more question: If I ever wanted to delete a winget project... Would it be enough to just delete the repo?
Sorry for the inconvenience!

I don't think it would worth it to upload that?
I think I should close that request and work on something better. That's why I asked how I close the pull request correctly.

Sorry for the inconveniences again

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @Basti-def,

Sorry to merge without addressing all of your concerns. To remove, please submit another PR with the manifest files removed. 

> I tested it on a virtual machine. You were right, I get the same output when I run this with the old powershell or cmd. It only works in the newer "Terminal" App.
> 
> I guess it have to work on both?

Feel free to update your application to work with the software that you'd like it to work with. I was just trying to present the minimum necessary to get the application to work on a Windows 10 VM. There's no requirement that the application be able to run on Terminal. This is why our manifests have so many fields and our processes have so many exceptions - because no 2 applications are alike. 

> Well actually, this is more like a "demo" project, to test if I can upload to winget. Can I close the commit, delete the repo and create a real + bigger project later, and start over?

Yes, what you choose to do with the application is your choice. You can close the commit, delete the repo, create another repo, add code to it, and submit a PR from that repo. You can have numerous software packages added at the same time, and can modify and life cycle those as well. 

> If that's all I have to do, I have one more question: If I ever wanted to delete a winget project... Would it be enough to just delete the repo?

Unfortunately, the process requires submitting another PR, as mentioned above. The winget-pkgs repo is something like a database, and so each change has to be performed through a merge. 

> I don't think it would worth it to upload that?

Software isn't judged on merit or complexity, only on functional operation. To pass the Automatic Pipeline, a package generally has to run for 10 seconds, and provide a zero exit code when closed. To pass the Manual Pipeline, packages have to operate normally for a human. 

> I think I should close that request and work on something better. That's why I asked how I close the pull request correctly.

Feel free to do as you please, within legal and moral guidelines. Please refer to the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/closing-a-pull-request) on closing PRs. 

> Sorry for the inconveniences again

Sorry to have completed the process without your consent.

**Pull request merged**